screen-orientation/onchange-event-subframe.html

```diff
@@ -8,6 +8,9 @@
 
 <script>
 promise_test(async t => {
+  await test_driver.bless("request full screen", () => {
```

Comment:
```suggestion
  await test_driver.bless("request fullscreen", () => {
```

Reply:
@marcoscaceres perhaps this suggestion is why it looked like there were conflicts. I'm not able to apply them and you probably can't either. If so just disregard it.